src/test/ui/macros/issue-54441.stderr

```diff
@@ -0,0 +1,14 @@
+error: expected one of `crate`, `fn`, `pub`, `static`, or `type`, found `let`
+  --> $DIR/issue-54441.rs:5:9
+   |
+LL | #![feature(macros_in_extern)]
+   | - expected one of `crate`, `fn`, `pub`, `static`, or `type` here
```

Comment:
Yes, this is the "expected" line pointing to line 1 of the source. However, it looks to be an existing bug in `Parser::expect_one_of()` WRT macros: https://play.rust-lang.org/?gist=1865a38a912b5aaed3e9a23931ace87c&version=stable&mode=debug&edition=2015

I can fix it here or in a different PR.

Reply:
>I can fix it here or in a different PR.

Whatever is more convenient for you.